setup.py

```diff
@@ -1,47 +1,13 @@
 from setuptools import setup
-from io import open
+from setuptools.config import read_configuration
+import os
 from kaitaistruct import __version__
 
-with open('README.rst', encoding='utf-8') as f:
-    long_description = f.read()
+this_dir=os.path.dirname(__file__)
+cfg = read_configuration(os.path.join(this_dir, 'setup.cfg'))
+cfg["metadata"]["version"]=__version__
+#print(cfg)
+cfg["options"].update(cfg["metadata"])
+cfg=cfg["options"]
```

Comment:
Please fix this broken formatting. Literally all style checkers (pep8, pycodestyle, pylint) complain about:

```
C:  6, 0: Exactly one space required around assignment
C:  8, 0: Exactly one space required around assignment
C: 11, 0: Exactly one space required around assignment
```

Probably optional:

```
C:  6, 0: Constant name "this_dir" doesn't conform to UPPER_CASE naming style (invalid-name)
C:  7, 0: Constant name "cfg" doesn't conform to UPPER_CASE naming style (invalid-name)
C: 11, 0: Constant name "cfg" doesn't conform to UPPER_CASE naming style (invalid-name)
C:  3, 0: standard import "import os" should be placed before "from setuptools import setup" (wrong-import-order)
```